### ISSUE TITLE
Error: no such column: rowid

### ISSUE THREAD

**Issue by anthonymclaughlin** [yy0931/sqlite3-editor#10]
Getting Error Message:

Error: no such column: rowid
Query: SELECT rowid AS rowid, * FROM "account_user" ORDER BY "rowid" LIMIT ? OFFSET ?
Params: [20, 0]

I used **WITHOUT ROWID** when I created this table:

```sql
CREATE TABLE IF NOT EXISTS "account_user" (
        ...
    ) WITHOUT ROWID;
```

Extension v1.0.50.

**Comment by yy0931** [comment]
I have an idea of the cause. I'll fix it in a minute.

**Comment by yy0931** [comment]
Fixed in v1.0.51.

I had forgotten that `.` in regex does not match newline characters.
